BREAKING.md

```diff
@@ -28,6 +29,9 @@ See [AgentScheduler-related deprecations](#AgentScheduler-related-deprecations)
 ### Changes to client-api
 - The `load` function from `document.ts` will fail the container does not exist. Going forward, please use the `create` function to handle this scenario.
 
+### DeltaManager.emitDelayInfo signature change
```

Comment:
DeltaManager class is not public API. We should not include this notice here

Reply:
Ok I was wondering if we expect anyone to access DeltaManager directly...  too bad we can't codify that somehow (or can we?)